package.json

```diff
@@ -364,6 +364,11 @@
           "type": "boolean",
           "description": "%java.debugger.configuration.enableHotCodeReplace.description%",
           "default": true
+        },
+        "java.debug.settings.enableRunDebugCodeLens": {
+          "type": "boolean",
+          "description": "Enable the run and debug code lens providers over main methods",
```

Comment:
use %template% style of the most recent commit

Reply:
Fixed. I used google translate to get the italian translation